### PR TITLE
DREF super ticket fixes

### PR DESCRIPTION
**Pull request by samshara**
## Addresses:
- https://github.com/IFRCGo/go-web-app/issues/379

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

**Review by UjjwalDhakal7 — APPROVED** [review]

**Review by roshni73 — APPROVED** [review]

**Review by utsukta1 — APPROVED** [review]

**Review by fereeca — APPROVED** [review]

**Review by roshni73 — APPROVED** [review]

File: src/views/AllDeployedPersonnel/index.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ import Table from '#components/Table';
 import DateInput from '#components/DateInput';
 import useFilterState from '#hooks/useFilterState';
 
+import { isDefined } from '@togglecorp/fujs';
```
Comment (barshathakuri) [inline src/views/AllDeployedPersonnel/index.tsx]: Please follow the coding structure.

File: src/views/AllDeployedPersonnel/index.tsx [hunk 2]
```diff
@@ -122,17 +123,25 @@ export function Component() {
...
                     || item.country_from?.name
                 ),
-                (item) => ({
-                    to: 'countriesLayout',
-                    urlParams: { countryId: item.country_from?.id },
-                }),
+                (item) => {
+                    if (isDefined(item.country_from?.record_type === 3)) {
+                        return {
+                            to: undefined,
```
Comment (barshathakuri) [inline src/views/AllDeployedPersonnel/index.tsx]: Why do we comparing `record_type` to 3. Use `enum`

File: src/views/DrefApplicationExport/styles.module.css [hunk 3]
```diff
@@ -228,4 +228,10 @@
             color: var(--go-ui-color-primary-red);
         }
     }
+
+    .targeting-strategy-link {
+        color: var(--go-ui-color-blue);
+        font-size: var(--go-ui-font-size-md);
+    }
+
```
Comment (barshathakuri) [inline src/views/DrefApplicationExport/styles.module.css]: Please Remove the extra line.

File: src/views/DrefApplicationForm/schema.ts [hunk 4]
```diff
@@ -316,13 +316,13 @@ const schema: DrefFormSchema = {
...
                     };
                 }
 
                 if (val?.type_of_dref === TYPE_IMMINENT) {
                     conditionalFields = {
                         ...conditionalFields,
-                        event_text: { validations: [max500CharCondition] },
                         anticipatory_actions: {},
                         supporting_document: {},
                     };
```
Comment (barshathakuri) [inline src/views/DrefApplicationForm/schema.ts]: Please revert the change `event_text: { validations: [max500CharCondition] },` should be inside the val?.type_of_dref === TYPE_IMMINENT condition.

File: src/views/EmergencySurge/RapidResponsePersonnelTable/index.tsx [hunk 5]
```diff
@@ -136,14 +136,22 @@ export default function RapidResponsePersonnelTable(props: Props) {
...
                 (item) => item.country_from?.society_name,
-                (item) => ({
-                    to: 'countriesLayout',
-                    urlParams: { countryId: item.country_from?.id },
-                }),
+                (item) => {
+                    if (isDefined(item.country_from?.record_type === 3)) {
+                        return {
+                            to: undefined,
+                        };
```
Comment (barshathakuri) [inline src/views/EmergencySurge/RapidResponsePersonnelTable/index.tsx]: Use `enums` for comparisons.

File: src/views/EmergencySurge/RapidResponsePersonnelTable/styles.module.css
Comment (barshathakuri) [inline src/views/EmergencySurge/RapidResponsePersonnelTable/styles.module.css:1]: Why did we rename this file?

File: src/views/EmergencySurge/RapidResponsePersonnelTable/i18n.json
Comment (barshathakuri) [inline src/views/EmergencySurge/RapidResponsePersonnelTable/i18n.json:1]: Why did we rename this file?

**Review by barshathakuri — CHANGES_REQUESTED** [review]

**Review by frozenhelium — APPROVED** [review]

**Review by frozenhelium — CHANGES_REQUESTED** [review]
We should squash the commits to avoid conflicts with the country pages @samshara

**Review by frozenhelium — APPROVED** [review]

**Review by barshathakuri — APPROVED** [review]

**Pull request merged**